pkg/ovsdb/client/client.go

```diff
@@ -59,7 +58,8 @@ func NewNbClient(addr string) (client.Client, error) {
 		}
 		options = append(options, client.WithEndpoint(ep))
 	}
-
+	ctx, cancel := context.WithTimeout(context.Background(), time.Duration(len(endpoints)+1)*timeout)
+	defer cancel()
```

Comment:
The context is not used in SSL, we should move it behind.

Reply:
ok